cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -230,7 +230,8 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 					snowbridge_pallet_system::Call::set_pricing_parameters { .. } |
 					snowbridge_pallet_system::Call::force_update_channel { .. } |
 					snowbridge_pallet_system::Call::force_transfer_native_from_agent { .. } |
-					snowbridge_pallet_system::Call::set_token_transfer_fees { .. },
+					snowbridge_pallet_system::Call::set_token_transfer_fees { .. } |
+					snowbridge_pallet_system::Call::force_register_token { .. }
```

Comment:
Why is just `force_register_token` added here and not `register_token`? Just so it is not enabled by default?

Reply:
Only `force_register_token` is used for assets on AH.

Reply:
Ah okay, so `force_register_token` is used for AH assets, and `register_token` for assets on other parachains? This is kind of related to this question here: https://github.com/Snowfork/polkadot-sdk/pull/128#discussion_r1559090024

Reply:
Exactly. Please also check my comment above for more details.